
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<HTML><HEAD><TITLE>Lee Sael</TITLE>
<META name="description" content="Computer Science, Seoul National University, Bio-Data Mining research and publications."> 
<META name="keywords" content="Sael Lee, Lee Sael, Seoul National University, Purdue University, Korea University, Computer Science, Bioinformatics, Computational Biology, Machine Learning, BioData Mining, Data Mining">
<!--
<meta name="verify-v1" content="G7qlYoXMQctkUxwJYts06trEj0KWKFyet69ACfB53QI=" >
-->
<META http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=robots content=index,nofollow>

<STYLE TYPE="text/css">

  	html, body	{	height: 100%; }
	
	body
	{
		margin: 10px;
		padding: 0px;
		background: #FFF;
		font-family: 'Open Sans', sans-serif;
		font-weight: 200;
		color: #2d2d2d;
	}
	

	
	h2
	{
		padding: 0.3em 0em 0.3em 0em;
		letter-spacing: 0.10em;
		text-transform: uppercase;
		font-size: 1.1em;
	}


   
</STYLE>

<script type="text/javascript">
onload = function(){
    document.getElementById("lastModified").innerHTML = "Last updated:" + document.lastModified.split(" ")[0];
}
</script>


</HEAD>

<div id="main">
<div id=class="container clearfix">

<BODY><A name=top></A>
<!-- BEGIN HEADER TABLE -->
<table  width="100%" class="yr_header">
    <tbody>
        <tr><td>
            <FONT size=2>
            <div class="network-links; height:30px;">
                <div style="float:left; padding:4px;"></div>
              <!--  <a href="http://www.sunykorea.ac.kr"><img src="image/SUNYK.jpg" alt="SUNYK" height=30></a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;				
                <a href="http://www.stonybrook.edu"><img src="image/SBU.jpg" alt="SBU" height=25></a>&nbsp; 
                <div style="float:right; padding:10px; margin-right:4px;" id='link-top'>
                <a href="http://www.cbbl.sunykorea.ac.kr">BioData Mining Lab</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
		        <a href="http://cs.sunykorea.ac.kr/">SUNYK CS</a>&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
                <a href="http://www.cs.sunysb.edu/">SBU CS</a>&nbsp;
-->
                </div>
        </div>
            </FONT>
	     </td></tr>
</tbody></table>
<table width="100%" height="20pt"><tbody>
	     <!-- banner --> 
         <tr><td align="right" bgcolor=#00539B><font size=2 color="#E6E6E6" > <span id="lastModified"></span>&nbsp;&nbsp;&nbsp;&nbsp;</font></td></tr>
    </tbody>
</table>
<!-- END HEADER TABLE -->
       

<!-- INTRO -->
	
<TABLE cellPadding=20 width=470 border=0>
  <TBODY>
  <TR vAlign=top>
    <TD><IMG src="image/sael.jpg" alt="Lee Sael" width=150 height=210><BR>
    <TD noWrap><STRONG><FONT size=+2></FONT></STRONG><FONT  
      size=5><B>Sael Lee (Lee Sael)</B></FONT><BR>
      BK Associate Professor<BR>
      <A HREF="http://cse.snu.ac.kr/">Dept. of Computer Science</A><BR>
      <A HREF="http://www.snu.ac.kr/">Seoul National University</A><BR><BR>

      <!--      <B>Lab: </B><a href="http://www.cbbl.sunykorea.ac.kr/">BioData Mining Lab</a><BR> --> 
      <B>Email:</B> saellee at snu.ac.kr <BR>
      <B>Office:</B> Seoul National University, Building 138, 2F<BR>
      1 Gwanak-ro, Gwanak-gu, Seoul, Korea 08826<BR>
      <BR> <BR>
    </TD>
  </TR>
  </TBODY>
</TABLE>


<!-- I hold a joint position as an assistant professor in the Department of Computer Science at SUNY Korea and an assistant research professor in the Department of Computer Science at Stony Brook University. -->  
I am currently a BK Associate Professor at Deptartment of Computer Science and Engineering at Seoul National University. 
I focus on solving various computational biology and bioinformatics problems via data-mining and machine learning approaches in addition to developing algorithms to improve clinical care and to gain deeper understanding of biology. 
Here is my short <A HREF="sael-bio.txt">biography</A> and full <A HREF="sael-cv.pdf">Curriculum Vitae</A>. <br>
(*<B>Biodata mining</B>: algorithms, systems, and discoveries for biological and clinical data analysis.) <br><br>

<b>NEWS</b>:<br>
<UL>
    <LI>[Nov. 15, 2017] I  gave a talk at Korea AI Society Workshop jointly held with <a href="http://www.acml-conf.org/2017/">ACML 2017</a> about "Making Tensor Factorization more Interpretable."
    <LI>[Nov. 2, 2017] I am invited to give a talk at <a href="https://www.kobic.kr/giw-bioinfo2017/">GIW/BIOINFO 2017</a> about "Scalable Tensor Factorization and Applications in Bioinformatics."  
    <LI>[Jun. 18-20, 2017] I gave a tutorial at 
        <a href="http://www.kiise.or.kr/conference/kcc/2017/">KCC2017</a> about
        <a href="talk/KCC2017/index.html">"Deep Learning in BioHealth"</a> with
        emphasis on Interpretability.
    <LI>[Feb. 16, 2017] I gave a tutorial at <a href="http://www.ksbi.or.kr/workshop/2017/biml2017_page.html/">BIML2017 </a> about <a href="talk/BIML2017/index.html"> "Mining and Learning Bio-Big Data" </a> introducing Scalable Matrix/Tensor Mining and Convolution Neural Network as tools for bio-data analyses. 

</UL>
<!-- END INTRO -->

<!-- BEGIN NAVIGATION TABLE -->
<div id="sidebar">
<div id="nav-anchor"> </div>
<nav>  
<table  width="100%"  height="20pt" >
    <tbody>
        <tr><td align="left" bgcolor=#00539B><font size=3 color="#E6E6E6" > 
		&nbsp;&nbsp;&nbsp;
	<!--<a href="#latest events" style="color:#e6e6e6">Latest Event</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp; -->
    <!--<a href="#courses" style="color:#e6e6e6">Courses</a> 
        &nbsp;&nbsp;&nbsp;|&nbsp;&nbsp; -->
	<a href="#research" style="color:#e6e6e6">Current Research</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#awards" style="color:#e6e6e6">Awards & Fellowships</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#current grants" style="color:#e6e6e6">Current Grants</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#talks" style="color:#e6e6e6">Talks</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
    <a href="#publications" style="color:#e6e6e6">Publications</a></a>&nbsp;
</font>
                </td></tr>
</tbody></table>
</nav>
<!-- END NAVIGATION TABLE -->

<!-- BEGIN CONTENT-->
<div id ="content">

<!--
<section id="announcement">
<P>
<H2>Announcements</H2>

<UL>
    <LI> I am looking for highly motivated Postdocs, Ph.D. and MS students. If interested, send me an email with your full Curriculum Vitae and transcript.
    <LI> I am also looking for undergraduate students who are interested in working on data mining. If interested, send me an email with your full Curriculum Vitae and transcript.
    <LI> Our lab holds the weekly <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar">Biodata Mining Seminar</A> which is open to all campus members interested in data mining and/or bioinformatics. 
</UL>
</section>


<section id="latest event">
<P>
<H2>Latest Events</H2>

<UL>
    <LI> (July 2015) Journal paper accepted in <a href="http://bioinformatics.oxfordjournals.org/content/early/2015/08/10/bioinformatics.btv409.abstract?keytype=ref&ijkey=z5iWUuNQQnz9MQj">Bioinformatics</a>, a top tier SCI Journal for Bioinformatics and Computational Biology.     
    <LI> (June 2015) New joint grant with Prof. Choi and Prof. De received on Machine Learning Business Intelligence.    
    <LI> (March 2015) New grant received on Big Bio-graph Analysis    
    <LI> (Dec 2014) A regular paper accepted to <a href="http://www.sigmod2015.org/">SIGMOD 2015</a>, a top tier database conference.
    <LI> (Aug 2014) A regular paper accepted to <a href="http://cikm2014.fudan.edu.cn/">CIKM 2014</a>, a top-tier data mining conference. 
</UL>
</section> 
-->
<!--
<section id="courses">
<P>
<H2>Courses</H2>

<UL>
<LI> CSE 214: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse214/">Computer Science II (Data Structures)</A>
<LI> CSE 305: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse305-532/">Principles of Database Systems </A>
<LI> CSE 373: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse373/">Analysis of Algorithms</A>
<LI> CSE 532: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse305-532/">Theory of Database Systems </A> 
<LI> CSE 537: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse537/">Artificial Intelligence</A>
<LI> CSE 548: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse548/">Analysis of Algorithms</A>
<LI> CSE 549: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse549/">Computational Biology</A>
<LI> CSE 600: <A HREF="http://cs.sunykorea.ac.kr/~cse600/index.html?sem=2016_spring">Topics in Modern Computer Science </A>
<LI> CSE 646: <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar/">Seminar on Artificial Intelligence (Seminar on Biodata Mining)</A>
</UL>

Course availability for this semester can be found <A HREF="http://www.cbbl.sunykorea.ac.kr/courses/">here</A>.
</section> --> <!-- courses -->


<section id="research">
<P>
<H2>Current Research </H2>
My primary research expands in the areas of <B>Bioinformatics</B>, <B>Data Mining</B>, and <B>Machine Learning</B>. I focus on development of methods and algorithms for mining and inference of biological data in various types (<B>variety</B>), which are either small or large (<B>volume</B>), with the emphasis on improved explainability (<B>interpretability</B>). I apply the methods to solve various precision medicine problems such as genomic profiling, cancer stratification, and disease gene prioritization.

<UL>
	<LI><I>Integrative analysis (Variety)</I>. 
	<UL>
		<LI>Heterogeneous data integration 
		<LI>Matrix/Tensor mining 
		<LI>Multiple kernel learning
	</UL>
	<LI> <I>Scalable methods (Volume)</I>
	<UL>
		<LI>Tensor decomposition   
		<LI>Graph ranking 
		<LI>L1 loss minimization 
	</UL>
	<LI><I>Explainable models (Interpretability) </I> 
	<UL>	<LI>Causal gene prioritization through protein structure 
		<LI>Pattern extraction of ICU signals via Convolution Neural Network 
		<LI>Graphical models
	</UL>
</UL>

<P>I also have expertise in the working with 3D protein structures with the focus of application on computation drug development. 

<UL>
	<LI>Protein structure search 
	<LI>Target detection & Adverse drug effects prediction
	<LI>Integrative structure prediction 
</UL>


<!--
<P>Please visit my lab page, i.e., <A HREF="http://www.cbbl.sunykorea.ac.kr">BioData Mining Lab</A> in the Department of Computer Science for details on  <A HREF="http://www.cbbl.sunykorea.ac.kr/research/">research</a>,  <A HREF="http://www.cbbl.sunykorea.ac.kr/"> latest academic events</A> and ongoing <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar"> lab seminar</A>.
<br>
</section> 
--> 
<!-- research interests-->

<section id="awards">
<P>
<H2> AWARDS & FELLOWSHIPS </H2>
<UL>
<LI>Best Paper Award (runner up), 6th International Conference on Emerging Databases (EDB), Oct. 19, 2016
<LI>Best Poster Award, International Conference on Structure Genomics 2011, May 14, 2011
<LI>Best Paper Award, The 21th International Conference on Genome Information (GIW). Dec. 18, 2010
<LI>Grace Hopper Scholarship sponsored by the National Science Foundation, 2008
</UL>
</section> <!-- awards-->


<section id="current grants">
<P>
<H2>Current Grants</H2>

<UL>
<LI>[2015.11-2018.10] PI: Basic Science Research Program, <B>"Development of Multi-layer Network Analysis Methods for Integrative Analysis Algorithms of Bio-Clinical Data"</B> (NRF-2015R1C1A2A01055739) Source of Award: the National Research Foundation of Korea (NRF) funded by the Ministry of Science, ICT and Future Planning (MSIP) Total Budget: 104,030,000 KRW (40,170,000 KRW 1st year) <BR><BR>

 <LI>[2015.06- 2018.02] PI: Global ATC Problem Development of business intelligence flatform service which enables a non-expert to realize <B>"Data Processing and User Interface within 3 seconds in Petabyte level based on 0.5 billion data (10053204)" </B>
    Source of Award: Korea Evaluation Institute of Industrial Technology, Republic of Korea
    Total Budget: 1,172,500,000KRW (175,000,000KRW/year);
    <BR><BR>

<LI>[2012.09-2018.02] Co-PI: <B> IT Consilience Creative Program NIPA-2013-H0203-13-1001 (ITCCP) </B>;
Source of Award: Ministry of Science, ICT and Future Planning (MSIP), Korea, supervised by the National IT Industry Promotion Agency (NIPA);
Total Budget (Government portion): 10,000,000,000 KRW (1,000,000,000 KRW/year)

</UL>

<H4>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*&nbsp;&nbsp;<A HREF="http://www3.cs.stonybrook.edu/~sael/past_grants.html">Completed Grant Projects</A></H4>
</section> <!-- current grants-->

<!--
<section id="software">

<H2>Software</H2>

<div>    <img SRC="pegasus_130.png" style="width:130px;height:54px;vertical-align:middle">    <span style=""> I lead the development of<A HREF="http://www.cs.cmu.edu/~pegasus" target="new">
 <B>PEGASUS (Peta-Scale Graph Mining System)</B></A>, an award-winning open-source software for mining very large graphs in Hadoop platform.</span> </div> 
</section> <!--software-->

<section id="talks">
<P>
<H2>Talks</H2>
<H4>Tutorials</H4>
    <UL>
        <LI> Sael Lee (Feb. 16, 2017), <a href="talk/BIML2017/index.html">Mining and Learning Bio-Big Data </a>
                at <a href="http://www.ksbi.or.kr/workshop/2017/biml2017_page.html/"> BIML2017</a>.
        <LI> Sael Lee (Jun. 18-20, 2017), <a href="talk/KCC2017/index.html">Deep Learning in BioHealth</a> at
 <a href="http://www.kiise.or.kr/conference/kcc/2017/">KCC2017</a>.
    </UL>
<!--H4>Invited Talks</H4-->

</section><!-- talk -->

<section id="publications">
<P>
<H2>Publications</H2>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; I try to keep the list of my publications up-to-date, however <A HREF="http://scholar.google.com/citations?user=mVPf6fQAAAAJ&hl=en">Google Scholar</A> does a better job. <BR>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <A HREF="http://dblp.uni-trier.de/pers/hd/s/Sael:Lee">DBLP</A> also lists publications in the computer science community but misses many with more biology flavor. 
<P>
<H4> 2017 </H4> 
<ul>
<li> Choi, D., & Sael, L. (2017). SNeCT: Scalable network constrained Tucker decomposition for integrative multi-platform data analysis. 
<A HREF="https://arxiv.org/abs/1711.08095">arXiv:1711.08095</A> <BR>
<A HREF="https://arxiv.org/pdf/1711.08095.pdf">[PDF]</A>
<A HREF="paper/1711.08095.html">[BIBTEX]</A>

<li> Thomas, J., Thomas, S. & Sael, L. (2017). Feature versus Raw Sequence: Deep Learning Comparative Study on Predicting Pre-miRNA. 
<A HREF="https://arxiv.org/abs/1710.06798">arXiv:1710.06798</A> <BR>
<A HREF="https://arxiv.org/pdf/1710.06798.pdf">[PDF]</A>
<A HREF="paper/1710.06798.html">[BIBTEX]</A>


<li> Lee, J., Choi, D., & Sael, L. (2017). CTD: Fast, Accurate, and Interpretable Method for Static and Dynamic Tensor Decompositions. 
<A HREF="https://arxiv.org/abs/1710.03608">arXiv:1710.03608</A> <BR>
<A HREF="https://arxiv.org/pdf/1710.03608.pdf">[PDF]</A>
<A HREF="paper/1710.03608.html">[BIBTEX]</A>


<li> Dehiya, V., Thomas, J., & Sael, L. (2017).  Prior Knowledge based mutation prioritization towards causal variant finding in rare disease.
<A HREF="https://arxiv.org/abs/1710.03399">arXiv:1710.03399</A> <BR>
<A HREF="https://arxiv.org/pdf/1710.03399.pdf">[PDF]</A>
<A HREF="paper/1710.03399.html">[BIBTEX]</A>

<li> Oh S., Park, N., Sael, L., Kang, U. (2017). Scalable Tucker Factorization for Sparse Tensors - Algorithms and Discoveries.  <A HREF="https://arxiv.org/abs/1710.02261">arXiv:1710.02261</A> <BR>
<A HREF="https://arxiv.org/pdf/1710.02261.pdf">[PDF]</A>
<A HREF="paper/1710.02261.html">[BIBTEX]</A>

<li>Thomas, J., & Sael, L. (2017). Deep Neural Network Based Precursor microRNA Prediction on Eleven Species. <A HREF="https://arxiv.org/abs/1704.03834">arXiv:1704.03834</A> <BR>
<A HREF="https://arxiv.org/pdf/1704.03834.pdf">[PDF]</A> 
<A HREF="paper/1704.03834.html">[BIBTEX]</A>

<li>Vo, Q. D., Thomas, J., Cho, S., De, P., Choi, B. J., & Sael, L. (2017). Next Generation Business Intelligence and Analytics: A Survey. <A HREF="https://arxiv.org/abs/1704.03402">arXiv:1704.03402</A> <BR>
<A HREF="https://arxiv.org/pdf/1704.03402.pdf">[PDF]</A> <A HREF="paper/1704.03402.html">[BIBTEX]</A>

<li>Thomas, J., & Sael, L. (2017). Multi-Kernel LS-SVM Based Integration Bio-Clinical Data Analysis and Application to Ovarian Cancer. International Journal of Data Mining and Bioinformatics. IJDMB, 19:(2), 150-167. <BR>
<A HREF="paper/Thomas, Sael - 2017 IJDMB.pdf">[PDF]</A> 
<A HREF="paper/Thomas, Sael - 2017 IJDMB.html">[BIBTEX]</A>

<li>Thomas, J., Thomas, S., & Sael, L. (2017). DP-miRNA: an improved prediction of precursor microRNA using deep learning mode. In IEEE International Conference on Big Data and Smart Computing (<a href="http://www.bigcomputing.org/conf2017/">IEEE BigComp 2017</a>) <BR>
<A HREF="paper/thomas, thomas, sael - 2017 - DP-miRNA.pdf">[PDF]</A>  <A HREF="paper/thomas2017.html">[BIBTEX]</A>  

<li>Shin, K., Sael, L., & Kang, U. (2017). Fully scalable methods for distributed tensor factorization. IEEE Transactions on Knowledge and Data Engineering (<B><a href="http://www.computer.org/portal/web/tkde">TKDE</a></B>), 29(1), 100-113. <BR>
<A HREF="paper/Shin, Sael, Kang - 2017 - CDTF.pdf">[PDF]</A>  <A HREF="paper/shin2017.html">[BIBTEX]</A>  

<li>Jung, J., Park, N., Sael, L., & Kang, U. (2017). BePI: Fast and memory-efficient method for billion-scale random walk with restart. In ACM International Conference on Management of Data (<B><a href="http://www.sigmod2017.org/">SIGMOD'17</a></B>). Raleigh, North Carolina, USA.: ACM Press.  <BR>
<A HREF="paper/Jung et al. - 2017 - BePI.pdf">[PDF]</A>  <A HREF="paper/jung2017.html">[BIBTEX]</A>  
</ul>

<P>
<H4> 2016 </H4> 
<ul>
<li>Thomas, J., & Sael, L. (2016). Maximizing information through multiple kernel-based heterogeneous data integration and applications to ovarian cancer. In 6th International Conference on Emerging Databases (<a href="http://www.emergingdb.org/edb2016">EDB</a>). ACM Press.<BR>
<A HREF="paper/Thomas, Sael - 2016 - Multiple kernel-based heterogeneous data integration.pdf">[PDF]</A>  <A HREF="paper/thomas2016b.html">[BIBTEX]</A> 

<li>Thomas, J., Seo, D., & Sael, L. (2016). Review on graph clustering and subgraph similarity based analysis of neurological disorders. International Journal of Molecular Sciences, 17(6), 862. <BR>
<A HREF="paper/Thomas, Seo, Sael - 2016 - Review on graph neurological disorders.pdf">[PDF]</A>  <A HREF="paper/thomas2016a.html">[BIBTEX]</A> 

<li>Jeon, B., Jeon, I., Sael, L., & Kang, U. (2016). SCouT: Scalable coupled matrix-tensor factorization - algorithm and discoveries. In 32nd IEEE International Conference on Data Engineering (<a href="http://icde2016.fi/"><B>ICDE2016</B></a>). Helsinki, Finland.<BR>
<A HREF="paper/Jeon et al. - 2016 - SCouT.pdf">[PDF]</A>
<A HREF="paper/jeon2016a.html">[BIBTEX]</A>

<li>Jeon, I., Papalexakis, E. E., Faloutsos, C., Sael, L., & Kang, U. (2016). Mining billion-scale tensors: algorithms and discoveries. The <B>VLDB</B> Journal. <BR>
<A HREF="paper/Jeon et al. - 2016 - Mining billion-scale tensors algorithms and discoveries.pdf">[PDF]</A>
<A HREF="paper/jeon2016b.html">[BIBTEX]</A>

<li>Jung, J., Jin, W., Sael, L., & Kang, U. (2016). Personalized ranking in signed networks using signed random walk with restart. In IEEE (Ed.), IEEE International Conference on Data Mining (<B><a href="http://icdm2016.eurecat.org/">ICDM</a></B>). Barcelona, Spain. <BR>
<A HREF="paper/Jung et al. - 2016 - Personalized ranking in signed networks using signed random walk with restart.pdf">[PDF]</A>
<A HREF="paper/jung2016a.html">[BIBTEX]</A>

<li>Jung, J., Shin, K., Sael, L., & Kang, U. (2016). Random walk with restart on large graphs using block elimination. ACM Transactions on Database Systems (<B>TODS</B>), 41(2), 1-43. <BR>
<A HREF="paper/Jung et al. - 2016 - Random walk with restart on large graphs using block elimination.pdf">[PDF]</A>
<A HREF="paper/jung2016b.html">[BIBTEX]</A>
</UL>


<P>
<H4> 2015 </H4> 
<UL>
<LI> Kim, S., Sael, L., & Yu, H. (2015). A mutation profile for top- k patient search exploiting Gene-Ontology and orthogonal non-negative matrix factorization. <B>Bioinformatics</B>, btv409. <br> 
        <A HREF="http://bioinformatics.oxfordjournals.org/content/early/2015/08/10/bioinformatics.btv409.full.pdf+html">[PDF]</A> 
        <A HREF="paper/kim2015a.html">[BIBTEX]</A>  
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/26209432">[PUBMED]</A>

 <LI> Shin, K., Jung, J., Sael, L., & Kang, U. (2015). BEAR: Block elimination approach for random walk with restart on large graphs. In Proceedings of the 2015 ACM SIGMOD International Conference on Management of Data (<a href="http://www.sigmod2015.org"><B>SIGMOD'15</B></a>). International Conference on Management of Data (pp. 1571-1585).  <BR>
        <A HREF="paper/Shin et al. - 2015 - BEAR.pdf">[PDF]</A> 
        <A HREF="paper/shin2015.html">[BIBTEX]</A> 
    
<LI> Sael, L., Jeon, I., & Kang, U. (2015). Scalable tensor mining. Big Data Research, 2(2), 82-86.  <br>
      <A HREF="sael, jeon, kang - 2015 - scalable tensor mining.pdf">[PDF]</A> 
        <A HREF="paper/Sael2015.html">[BIBTEX]</A>
      
<LI> Thomas, J., & Sael, L. (2015). Overview of integrative analysis methods for heterogeneous data. In The 2015 International Conference on Big Data and Smart Computing (<a href="http://www.bigcomputing.org/conf2015">BigComp 2015</a>) (pp. 266-270). <br>
        <A HREF="paper/Thomas, Sael - 2015 - Overview of integrative analysis methods for heterogeneous data.pdf">[PDF]</A> 
        <A HREF="paper/Thomas2015.html">[BIBTEX]</A>  
        <!--A HREF="">[PUBMED]</A-->

<LI> Kim, S., Sael, L., & Yu, H. (2015). LMDS-based approach for efficient top-k local ligand-binding site search. International Journal of Data Mining and Bioinformatics, 12(4), 417-433. <br>
        <A HREF="paper/kim, sael, yu - 2015 - lmds-based approach for efficient top-k local ligand-binding site search.pdf">[PDF]</A> 
        <A HREF="paper/kim2015.html">[BIBTEX]</A> 
</UL>


<H4>2014</H4>
<UL>
<LI> Kang, D., Lim, W., Shin, K., Sael, L., & Kang, U. (2014). Data/Feature Distributed Stochastic Coordinate Descent for Logistic Regression. In Proceedings of the 23rd International Conference on Information and Knowledge Management (<B><a href="http://cikm2014.fudan.edu.cn/">CIKM 2014</a></B>). Shanghai, China: ACM Press. <BR>
        <A HREF="paper/kang et al. - 2014 - data feature distributed stochastic coordinate descent for logistic regression.pdf">[PDF]</A> 
        <A HREF="paper/dfdscd_supp_theory.pdf">[Supplementary Document]</A> 
        <A HREF="paper/kang2014.html">[BIBTEX]</A>
 
<LI>Xiong, Y., Esquivel-Rodriguez, J., Sael, L., & Kihara, D. (2014). 3D-SURFER 2.0: Web platform for real-time search and characterization of protein surfaces. Methods in Molecular Biology, 1137, 105-117.<BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/xiong2014.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/24573477">[PUBMED]</A>
 
<LI> Kim, S., Sael, L., & Yu, H. (2014). Identifying cancer subtypes based on somatic mutation profile. In Proceedings of the ACM 8th International Workshop on Data and Text Mining in Bioinformatics - <a href="http://www.dtmbio.net/dtmbio2014/">DTMBIO2014</a> (pp. 19-22). New York, NY, USA: ACM.  <BR>
        <A HREF="paper/Kim, Sael, Yu - 2014 - Identifying cancer subtypes based on somatic mutation profile.pdf">[PDF]</A> 
        <A HREF="paper/Kim2014.html">[BIBTEX]</A> 
</UL> 


<H4>2013</H4>
<UL>
<LI> Pi, J., & Sael, L. (2013). Mass spectrometry coupled experiments and protein structure modeling methods. International Journal of Molecular Sciences, 14(10), 20635-20657. <BR>
        <A HREF="http://www.mdpi.com/1422-0067/14/10/20635/pdf">[PDF]</A>
        <A HREF="paper/Pi2013.html">[BIBTEX]</A>         <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/24132151">[PUBMED]</A>
        
<LI> Kim, S., Sael, L., & Yu, H. (2013). Efficient protein structure search using indexing methods. BMC Medical Informatics and Decision Making, 13(Suppl 1), S8. <BR>
        <A HREF="http://www.biomedcentral.com/content/pdf/1472-6947-13-S1-S8.pdf">[PDF]</A>
        <A HREF="paper/Kim2013d.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/23691543">[PUBMED]</A>
         
<LI>Tang, M., Tan, K. M., Tan, X. L., Sael, L., Chitale, M., Esquivel-rodriguez, J., & Kihara, D. (2014). Graphical models for protein function and structure prediction. In M. Elloumi & A. Y. Zomaya (Eds.), Biological Knowledge Discovery Handbook: Preprocessing, Mining, and Postprocessing of Biological Data (pp. 191-222). Hoboken, New Jersey: John Wiley & Sons, Inc. <BR>
        <A HREF="http://onlinelibrary.wiley.com/doi/10.1002/9781118617151.ch09/pdf">[PDF]</A> 
        <A HREF="paper/Tang2013.html">[BIBTEX]</A>
 
 <LI> Coelho, D., & Sael, L. (2013). Breast and prostate cancer expression similarity analysis by iterative SVM based ensemble gene selection. In Proceedings of the 7th International Workshop on Data and Text Mining in Biomedical Informatics - <a href="http://www.dtmbio.net/dtmbio2013/">DTMBIO'13</a> (pp. 23-26). San Francisco, USA: ACM Press. 
        <A HREF="paper/Coelho, Sael - 2013 - Breast and prostate cancer expression similarity analysis by iterative SVM based ensembl.pdf">[PDF]</A>
        <A HREF="paper/Coelho2013.html">[BIBTEX]</A>     

<LI>Kim, S., Sael, L., & Yu, H. (2013). Efficient local ligand-binding site search using landmark MDS. In Proceedings of the 7th International Workshop on Data and Text Mining in Biomedical Informatics - <a href="http://www.dtmbio.net/dtmbio2013/">DTMBIO'13</a> (pp. 13-14). San Francisco, USA: ACM Press.<BR>
        <A HREF="paper/Kim, Sael, Yu - 2013 - Efficient local ligand-binding site search using landmark mds.pdf">[PDF]</A>
         <A HREF="paper/Kim2013b.html">[BIBTEX]</A>

<LI> Kim, S., Sael, L., & Yu, H. (2013). Fast protein 3D surface search. In Proceedings of the 7th International Conference on Ubiquitous Information Management and Communication - <a href="http://www.icuimc.org/2013/index.html">ICUIMC'13 </a>(pp. 1-8). Kota Kinabalu, Malaysia: ACM Press. <BR>
         <A HREF="paper/Kim, Sael, Yu - 2013 - Fast protein 3D surface search.pdf">[PDF]</A> 
        <A HREF="paper/Kim2013a.html">[BIBTEX]</A>         
   
</UL>

<H4>2012</H4>
<UL>
<LI> Mullins, E. a, Starks, C. M., Francois, J. a, Sael, L., Kihara, D., & Kappock, T. J. (2012). Formyl-coenzyme A (CoA):oxalate CoA-transferase from the acidophile Acetobacter aceti has a distinctive electrostatic surface and inherent acid stability. <B>Protein Science</B>, 21(5), 686-696. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Mullins2012.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22374910">[PUBMED]</A>

<LI>Sael, L., Chitale, M., & Kihara, D. (2012). Structure- and sequence-based function prediction for non-homologous proteins. Journal of Structural and Functional Genomics, 13(2), 111-23. <BR>
 <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012b.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22270458">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2012). Detecting local ligand-binding site similarity in nonhomologous proteins by surface patch comparison. <B>Proteins</B>: Structure, Function, and Bioinformatics, 80(4), 1177-1195. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22275074">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2012). Constructing patch-based ligand-binding pocket database for predicting function of proteins. <B>BMC Bioinformatics</B>, 13(Suppl 2), S7. 
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012a.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22536870">[PUBMED]</A>
     
<LI>Kim, S., Lee, S., & Yu, H. (2012). Indexing methods for efficient protein 3D surface search. In Proceedings of the ACM sixth international workshop on Data and text mining in biomedical informatics - <a href="http://www.dtmbio.net/dtmbio2012/> DTMBIO'12</a> (pp. 41-48). <BR>
         <A HREF="paper/kim, sael, yu - 2012 - indexing methods for efficient protein 3d surface search.pdf">[PDF]</A> 
         <A HREF="paper/Kim2012.html">[BIBTEX]</A>
</UL>   

<H4>2011</H4>
<UL>
    <LI> Kihara, D., Sael, L., Chikhi, R., & Esquivel-Rodriguez, J. (2011). Molecular surface representation using 3D Zernike descriptors for protein shape comparison and docking. Current Protein & Peptide Science, 12(6), 520-530. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Kihara2011.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21787306">[PUBMED]</A>
    
<LI> Sael, L. Structural bioinformatics and application to future drug design evolution. Communications of KIISE. 2011; 29:4, 26-32. <BR>
        <!-- <A HREF="">[PDF]</A> --> 

<LI> Chikhi, R., Sael, L., & Kihara, D. Protein binding ligand prediction using moment-based methods  In D. Kihara (Ed.), Protein Function Prediction for Omics Era (pp. 145=163). Dordrecht: Springer Netherlands. 2011. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Chikhi2011.html">[BIBTEX]</A> 
        <A href="http://www.springer.com/biomed/book/978-94-007-0880-8">[ABSTRACT]</A>    

</UL>

<H4>2010</H4>
<UL>
 <LI> Chikhi, R., Sael, L., & Kihara, D. (2010). Real-time ligand binding pocket database search using local surface descriptors. <B>Proteins</B>: Structure, Function, and Bioinformatics, 78(9), 2007-28. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Chikhi2010.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20455259">[PUBMED]</A>
    
<LI> Sael, L., & Kihara, D. (2010). Binding ligand prediction for proteins using partial matching of local surface patches. International Journal of Molecular Sciences, 11, 5009-5026. <BR>
        <A HREF="http://www.mdpi.com/1422-0067/11/12/5009/pdf">[PDF]</A> 
        <A HREF="paper/Sael2010a.html">[BIBTEX]</A>         <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21614188">[PUBMED]</A>
    
<LI> Sael, L., & Kihara, D. (2010). Improved protein surface comparison and application to low-resolution protein structure data. <B>BMC Bioinformatics</B>, 11(Suppl 1), S2. (GIW2010 - BEST PAPER AWARD)  <BR>
        <A HREF="http://www.biomedcentral.com/content/pdf/1471-2105-11-S11-S2.pdf">[PDF]</A>
        <A HREF="paper/Sael2010.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21172052">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2010). Characterization and classification of local protein surfaces using self-organizing map. International Journal of Knowledge Discovery in Bioinformatics, 1(1), 32-47. <BR>
        <A HREF="paper/SaelKihara_IJKDB.pdf">[PDF]</A>
        <A HREF="paper/Sael2010b.html">[BIBTEX]</A>
        <A target="blank" href="http://www.igi-global.com/Bookstore/Article.aspx?TitleId=40970">[ABSTRACT]</A>
</UL>

<H4>2009</H4>
<UL>
<LI> La, D., Esquivel-rodriguez, J., Venkatraman, V., Li, B., Sael, L., Ueng, S., Steven, A., Kihara, D. (2009). 3D-SURFER: Software for high-throughput protein surface comparison and analysis. <B>Bioinformatics</B>, 25(21), 2843-4. <BR>
        <A HREF="http://bioinformatics.oxfordjournals.org/content/25/21/2843.full.pdf+html">[PDF]</A> 
        <A HREF="paper/La2009.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/19759195">[PUBMED]</A>

<LI> Venkatraman, V., Yang, Y. D., Sael, L., & Kihara, D. (2009). Protein-protein docking using region-based 3D Zernike descriptors. <B>BMC Bioinformatics</B>, 10, 407. <BR>
        <A href="http://www.biomedcentral.com/content/pdf/1471-2105-10-407.pdf">[PDF]</A>
        <A HREF="paper/Venkatraman2009.html">[BIBTEX]</A>
        <A href="paper/lzerd_Suppl.pdf">[Suppl. Data]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20003235">[PUBMED]</A>
    
<LI>Venkatraman, V., Sael, L., & Kihara, D. (2009). Potential for protein surface shape analysis using spherical harmonics and 3D Zernike descriptors. Cell Biochemistry and Biophysics, 54(1-3), 23-32. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Venkatraman2009a.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/19521674">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2009). Protein surface representation and comparison: New approaches in structural proteomics. In J. Y. Chen & S. Lonardi (Eds.), Biological Data Mining (pp. 89-109). USA: Chapman & Hall/CRC Press. <BR>
        <A HREF="paper/Sael, Kihara - Biological Data Mining 2009.pdf">[PDF]</A> 
       <A HREF="paper/Sael2009.html">[BIBTEX]</A> 
        <A href="http://www.amazon.com/Biological-Mining-Chapman-Knowledge-Discovery/dp/1420086847/">[ABSTRACT]</A>    

</UL>
<H4>2008</H4>
<UL>
<LI> Sael, L., La, D., Li, B., Rustamov, R., & Kihara, D. (2008). Rapid comparison of properties on protein surface. <B>Proteins</B>: Structure, Function, and Bioinformatics, 73(1), 1-10.  <BR>
        <!-- <A HREF="">[PDF]</A> --> 
        <A HREF="paper/Sael2008.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/18618695">[PUBMED]</A>
    
<LI>Sael, L., Li, B., La, D., Fang, Y., Ramani, K., Rustamov, R., & Kihara, D. (2008). Fast protein tertiary structure retrieval based on global surface shape similarity. <B>Proteins</B>: Structure, Function, and Bioinformatics, 72(4), 1259-1273. <BR>
        <!-- <A HREF="">[PDF]</A> --> 
        <A HREF="paper/Sael2008a.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/18361455">[PUBMED]</A>
        <BR>
        <BR>
</UL>
</section> <!-- publications -->
<P>


<!-- END CONTENT-->

</div> <!-- /#content -->
</div> <!-- /.container -->
</div> <!-- /#main -->

<table width="100%" height="30pt"><tbody>
        <tr><td align="left" bgcolor=#00539B>
        <FONT size=2 color="white">
            <div style="float:left; padding:5px;"></div> 
        Seoul National University, 1 Gwanak-ro, Gwanak-gu, Seoul, Korea 08826
        </FONT>
        </td></tr>
    </tbody>
</table>

<!-- Start of StatCounter Code for Default Guide -->
<script type="text/javascript">
var sc_project=10114058; 
var sc_invisible=1; 
var sc_security="19da72e3"; 
var scJsHost = (("https:" == document.location.protocol) ?
"https://secure." : "http://www.");
document.write("<sc"+"ript type='text/javascript' src='" +
scJsHost+
"statcounter.com/counter/counter.js'></"+"script>");
</script>
<noscript><div class="statcounter"><a title="web stats"
href="http://statcounter.com/" target="_blank"><img
class="statcounter"
src="http://c.statcounter.com/10114058/0/19da72e3/1/"
alt="web stats"></a></div></noscript>
<!-- End of StatCounter Code for Default Guide -->

</BODY></HTML>










