
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
<!--html lang="ko"-->
<HTML><HEAD>


<TITLE>Lee Sael</TITLE>


<!--HEADER INCLUDE-->
<object name="header" width="100%" height="80pt" type="text/html" data="header.html"></object>

<META name="keywords" content="Sael Lee, Lee Sael, Seoul National University, Purdue University, Korea University, Computer Science, Bioinformatics, Computational Biology, Machine Learning, BioData Mining, Data Mining, Interpretable Machine Learning, 이슬, 서울대학교, 이슬교수, 컴퓨터미래인재양성사업단, Seoul National University, Bioinformatics, Data Mining, Machine Learning, Biohealth, Data Science, Purdue, Korea University ">
<META http-equiv=Content-Type content="text/html; charset=utf-8">
<meta name=robots content=index>
<meta name="description" content='academic personal webpage of professor Sael Lee AKA Lee Sael'>

<link rel="stylesheet" type="text/css" href="mystyle.css">


</HEAD>



<!-- INTRO -->
<BODY>
<TABLE cellPadding=20 width=470 border=0>
  <TBODY>
  <TR vAlign=top>
    <TD><IMG src="image/sael.jpg" alt="Lee Sael" width=150 height=210><BR>
    <TD noWrap><STRONG><FONT size=+2></FONT></STRONG><FONT  
      size=5><B>Sael Lee (Lee Sael)</B></FONT><BR>
      BK Associate Professor<BR>
      <A HREF="http://cse.snu.ac.kr/">Dept. of Computer Science and Engineering</A><BR>
      <A HREF="http://www.snu.ac.kr/">Seoul National University</A><BR><BR>

      <!--      <B>Lab: </B><a href="http://www.cbbl.sunykorea.ac.kr/">BioData Mining Lab</a><BR> --> 
      <B>Email:</B> saellee at snu.ac.kr <BR>
      <B>Office:</B> Seoul National University, Building 138, 2F<BR>
      1 Gwanak-ro, Gwanak-gu, Seoul, Korea 08826<BR>
      <BR> <BR>
    </TD>
  </TR>
  </TBODY>
</TABLE>


<!-- I hold a joint position as an assistant professor in the Department of Computer Science at SUNY Korea and an assistant research professor in the Department of Computer Science at Stony Brook University. -->  
I am currently a BK Associate Professor at Deptartment of Computer Science and Engineering at Seoul National University. 
I focus on solving various computational biology and bioinformatics problems via data-mining and machine learning approaches in addition to developing algorithms to improve clinical care and to gain deeper understanding of biology. 
Here is my short <A HREF="sael-bio.txt">biography</A> and full <A HREF="sael-cv.pdf">Curriculum Vitae</A>. <br>
(*<B>Biodata mining</B>: algorithms, systems, and discoveries for biological and clinical data analysis.) <br><br>

<b>NEWS</b>:<br>
<UL>
    <LI>[Jun. 18-20, 2017] I gave a tutorial at 
        <a href="http://www.kiise.or.kr/conference/kcc/2017/">KCC2017</a> about
        <a href="talk/KCC2017/index.html">"Deep Learning in BioHealth"</a> with
        emphasis on Interpretability.
    <LI>[Feb. 16, 2017] I gave a tutorial at <a href="http://www.ksbi.or.kr/workshop/2017/biml2017_page.html/">BIML2017 </a> about <a href="talk/BIML2017/index.html"> "Mining and Learning Bio-Big Data" </a> introducing Scalable Matrix/Tensor Mining and Convolution Neural Network as tools for bio-data analyses. 

</UL>
<!-- END INTRO -->

<!-- BEGIN NAVIGATION TABLE -->
<div id="sidebar">
<div id="nav-anchor"> </div>
<nav>  
<table  width="100%"  height="20pt" >
    <tbody>
        <tr><td align="left" bgcolor=#00539B><font size=3 color="#E6E6E6" > 
		&nbsp;&nbsp;&nbsp;
	<!--<a href="#latest events" style="color:#e6e6e6">Latest Event</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp; -->
    <!--<a href="#courses" style="color:#e6e6e6">Courses</a> 
        &nbsp;&nbsp;&nbsp;|&nbsp;&nbsp; -->
	<a href="#research" style="color:#e6e6e6">Current Research</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#awards" style="color:#e6e6e6">Awards & Fellowships</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#current grants" style="color:#e6e6e6">Current Grants</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
	<a href="#talks" style="color:#e6e6e6">Talks</a>
		&nbsp;&nbsp;&nbsp;|&nbsp;&nbsp;
    <a href="#publications" style="color:#e6e6e6">Publications</a></a>&nbsp;
</font>
                </td></tr>
</tbody></table>
</nav>
<!-- END NAVIGATION TABLE -->

<!-- BEGIN CONTENT-->
<div id ="content">

<!--
<section id="announcement">
<P>
<H2>Announcements</H2>

<UL>
    <LI> I am looking for highly motivated Postdocs, Ph.D. and MS students. If interested, send me an email with your full Curriculum Vitae and transcript.
    <LI> I am also looking for undergraduate students who are interested in working on data mining. If interested, send me an email with your full Curriculum Vitae and transcript.
    <LI> Our lab holds the weekly <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar">Biodata Mining Seminar</A> which is open to all campus members interested in data mining and/or bioinformatics. 
</UL>
</section>


<section id="latest event">
<P>
<H2>Latest Events</H2>

<UL>
    <LI> (July 2015) Journal paper accepted in <a href="http://bioinformatics.oxfordjournals.org/content/early/2015/08/10/bioinformatics.btv409.abstract?keytype=ref&ijkey=z5iWUuNQQnz9MQj">Bioinformatics</a>, a top tier SCI Journal for Bioinformatics and Computational Biology.     
    <LI> (June 2015) New joint grant with Prof. Choi and Prof. De received on Machine Learning Business Intelligence.    
    <LI> (March 2015) New grant received on Big Bio-graph Analysis    
    <LI> (Dec 2014) A regular paper accepted to <a href="http://www.sigmod2015.org/">SIGMOD 2015</a>, a top tier database conference.
    <LI> (Aug 2014) A regular paper accepted to <a href="http://cikm2014.fudan.edu.cn/">CIKM 2014</a>, a top-tier data mining conference. 
</UL>
</section> 
-->
<!--
<section id="courses">
<P>
<H2>Courses</H2>

<UL>
<LI> CSE 214: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse214/">Computer Science II (Data Structures)</A>
<LI> CSE 305: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse305-532/">Principles of Database Systems </A>
<LI> CSE 373: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse373/">Analysis of Algorithms</A>
<LI> CSE 532: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse305-532/">Theory of Database Systems </A> 
<LI> CSE 537: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse537/">Artificial Intelligence</A>
<LI> CSE 548: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse548/">Analysis of Algorithms</A>
<LI> CSE 549: <A HREF="http://www.cs.stonybrook.edu/~sael/teaching/cse549/">Computational Biology</A>
<LI> CSE 600: <A HREF="http://cs.sunykorea.ac.kr/~cse600/index.html?sem=2016_spring">Topics in Modern Computer Science </A>
<LI> CSE 646: <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar/">Seminar on Artificial Intelligence (Seminar on Biodata Mining)</A>
</UL>

Course availability for this semester can be found <A HREF="http://www.cbbl.sunykorea.ac.kr/courses/">here</A>.
</section> --> 
<!-- courses -->


<section id="research">
<P>
<H2>Current Research </H2>
My primary research expands in the areas of <B>Bioinformatics</B>, <B>Data Mining</B>, and <B>Machine Learning</B>. I focus on development of methods and algorithms for mining and inference of biological data in various types (<B>variety</B>), which are either small or large (<B>volume</B>), with the emphasis on improved explainability (<B>interpretability</B>). I apply the methods to solve various precision medicine problems such as genomic profiling, cancer stratification, and disease gene prioritization.

<UL>
	<LI><I>Integrative analysis (Variety)</I>. 
	<UL>
		<LI>Heterogeneous data integration 
		<LI>Matrix/Tensor mining 
		<LI>Multiple kernel learning
	</UL>
	<LI> <I>Scalable methods (Volume)</I>
	<UL>
		<LI>Tensor decomposition   
		<LI>Graph ranking 
		<LI>L1 loss minimization 
	</UL>
	<LI><I>Explainable models (Interpretability) </I> 
	<UL>	<LI>Causal gene prioritization through protein structure 
		<LI>Pattern extraction of ICU signals via Convolution Neural Network 
		<LI>Graphical models
	</UL>
</UL>

<P>I also have expertise in the working with 3D protein structures with the focus of application on computation drug development. 

<UL>
	<LI>Protein structure search 
	<LI>Target detection & Adverse drug effects prediction
	<LI>Integrative structure prediction 
</UL>


<!--
<P>Please visit my lab page, i.e., <A HREF="http://www.cbbl.sunykorea.ac.kr">BioData Mining Lab</A> in the Department of Computer Science for details on  <A HREF="http://www.cbbl.sunykorea.ac.kr/research/">research</a>,  <A HREF="http://www.cbbl.sunykorea.ac.kr/"> latest academic events</A> and ongoing <A HREF="http://www.cbbl.sunykorea.ac.kr/seminar"> lab seminar</A>.
<br>
</section> 
--> 
<!-- research interests-->

<section id="awards">
<P>
<H2> AWARDS & FELLOWSHIPS </H2>
<UL>
<LI>Best Paper Award (runner up), 6th International Conference on Emerging Databases (EDB), Oct. 19, 2016
<LI>Best Poster Award, International Conference on Structure Genomics 2011, May 14, 2011
<LI>Best Paper Award, The 21th International Conference on Genome Information (GIW). Dec. 18, 2010
<LI>Grace Hopper Scholarship sponsored by the National Science Foundation, 2008
</UL>
</section> <!-- awards-->


<section id="current grants">
<P>
<H2>Current Grants</H2>

<UL>
<LI>[2018.09-2021.08] PI: Basic Science Research Program, "<B>Interpretable and Dynamic Tensor Decomposition Method Development and Applications to Personalized Clinical Analysis</B> (NRF-2018R1A1A3A0407953),” supported by the National Research Foundation of Korea (NRF).
<LI>[2018.08-2025.02] Co-PI: Engineering Research Center (ERC), "<B>Extreme Exploitation of Dark Data </B>(NRF-2018R1A5A1060031),” supported by the National Research Foundation of Korea (NRF).
<LI>[2015.11-2018.10] PI: Basic Science Research Program, "<B>Development of Multi-layer Network Analysis Methods for Integrative Analysis Algorithms of Bio-Clinical Data</B> (NRF-2015R1C1A2A01055739)," supported by the the National Research Foundation of Korea (NRF).
</UL>

<H4>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*&nbsp;&nbsp;<A HREF="past_grants.html">Completed Grant Projects</A></H4>
</section> <!-- current grants-->

<!--
<section id="software">

<H2>Software</H2>

<div>    <img SRC="pegasus_130.png" style="width:130px;height:54px;vertical-align:middle">    <span style=""> I lead the development of<A HREF="http://www.cs.cmu.edu/~pegasus" target="new">
 <B>PEGASUS (Peta-Scale Graph Mining System)</B></A>, an award-winning open-source software for mining very large graphs in Hadoop platform.</span> </div> 
</section> <!--software-->

<section id="talks">
<P>
<H2>Talks</H2>
<H4>Tutorials</H4>
    <UL>
        <LI> Sael Lee (Feb. 16, 2017), <a href="talk/BIML2017/index.html">Mining and Learning Bio-Big Data </a>
                at <a href="http://www.ksbi.or.kr/workshop/2017/biml2017_page.html/"> BIML2017</a>.
        <LI> Sael Lee (Jun. 18-20, 2017), <a href="talk/KCC2017/index.html">Deep Learning in BioHealth</a> at
 <a href="http://www.kiise.or.kr/conference/kcc/2017/">KCC2017</a>.
    </UL>

<H4>Invited Talks</H4>
    <UL>
    <LI>Sael Lee (Nov. 15, 2017) "Making Tensor Factorization more Interpretable." at Korea AI Society Workshop jointly held with <a href="http://www.acml-conf.org/2017/">ACML 2017</a>. 
    <LI>Sael Lee (Nov. 2, 2017) "Scalable Tensor Factorization and Applications in Bioinformatics" at <a href="https://www.kobic.kr/giw-bioinfo2017/">GIW/BIOINFO 2017</a>.
    </UL>


</section><!-- talk -->

<section id="publications">
<P>
<H2>Publications</H2>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; I try to keep the list of my publications up-to-date, however <A HREF="http://scholar.google.com/citations?user=mVPf6fQAAAAJ&hl=en">Google Scholar</A> does a better job. <BR>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; <A HREF="http://dblp.uni-trier.de/pers/hd/s/Sael:Lee">DBLP</A> also lists publications in the computer science community but misses many with more biology flavor. 
<P>
<H4> 2018 </H4> 
<ul>

<li> Lee, J., Oh S., Sael, L. (2018). GIFT: Guided and Interpretable Factorization for Tensors with an Application to Large-Scale Multi-platform Cancer Analysis. 
<B>Bioinformatics</B>, bty490.<BR>
        <A HREF="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty490/5042168?guestAccessKey=5e8a16c1-d34b-48a3-b77a-f8b96144eb22">[PDF]</A>
        <<A HREF="paper/lee2018b.html">[BIBTEX]</A>-->
        <A HREF="https://github.com/leesael/GIFT">[GITHUB]</A>
     
<li> Lee, J., Choi, D., & Sael, L. (2018). CTD: Fast, Accurate, and Interpretable Method for Static and Dynamic Tensor Decompositions. 
PLOS One 13(7):e0200579. <BR>
        <A HREF="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0200579&type=printable">[PDF]</A>
        <<A HREF="paper/lee2018.html">[BIBTEX]</A>
        <A HREF="https://github.com/leesael/CTD">[GITHUB]</A>

<li> Oh S., Park, N., Sael, L., Kang, U. (2018). Scalable Tucker Factorization for Sparse Tensors - Algorithms and Discoveries.  
34th IEEE International Conference on Data Engineering (<A HREF="https://icde2018.org/">ICDE</A>)  2018, Paris, France.<BR>
        <A HREF="https://datalab.snu.ac.kr/ptucker/ptucker.pdf">[PDF]</A>
        <!--<A HREF="paper/1710.02261.html">[BIBTEX]</A>-->
        <A HREF="https://datalab.snu.ac.kr/ptucker/supple.pdf">[SUP]</A>

<li> Dehiya, V., Thomas, J., & Sael, L. (2017).  Prior Knowledge based mutation prioritization towards causal variant finding in rare disease.
 PLOS One (accepted)<BR>
        <A HREF="https://arxiv.org/pdf/1710.03399.pdf">[PDF]</A>
        <!--A HREF="paper/1710.03399.html">[BIBTEX]</A-->        

</ul>

<P>
<H4> 2017 </H4> 
<ul>
<li> Choi, D., & Sael, L. (2017). SNeCT: Scalable network constrained Tucker decomposition for integrative multi-platform data analysis. 
        <A HREF="https://arxiv.org/abs/1711.08095">arXiv:1711.08095</A> <BR>
        <A HREF="https://arxiv.org/pdf/1711.08095.pdf">[PDF]</A>
        <A HREF="paper/1711.08095.html">[BIBTEX]</A>

<li> Thomas, J., Thomas, S. & Sael, L. (2017). Feature versus Raw Sequence: Deep Learning Comparative Study on Predicting Pre-miRNA. 
        <A HREF="https://arxiv.org/abs/1710.06798">arXiv:1710.06798</A> <BR>
        <A HREF="https://arxiv.org/pdf/1710.06798.pdf">[PDF]</A>
        <A HREF="paper/1710.06798.html">[BIBTEX]</A>

<li>Thomas, J., & Sael, L. (2017). Deep Neural Network Based Precursor microRNA Prediction on Eleven Species. <A HREF="https://arxiv.org/abs/1704.03834">arXiv:1704.03834</A> <BR>
        <A HREF="https://arxiv.org/pdf/1704.03834.pdf">[PDF]</A> 
        <A HREF="paper/1704.03834.html">[BIBTEX]</A>

<li>Vo, Q. D., Thomas, J., Cho, S., De, P., Choi, B. J., & Sael, L. (2017). Next Generation Business Intelligence and Analytics: A Survey. <A HREF="https://arxiv.org/abs/1704.03402">arXiv:1704.03402</A> <BR>
        <A HREF="https://arxiv.org/pdf/1704.03402.pdf">[PDF]</A> 
        <A HREF="paper/1704.03402.html">[BIBTEX]</A>

<li>Thomas, J., & Sael, L. (2017). Multi-Kernel LS-SVM Based Integration Bio-Clinical Data Analysis and Application to Ovarian Cancer. International Journal of Data Mining and Bioinformatics. IJDMB, 19:(2), 150-167. <BR>
        <A HREF="paper/Thomas, Sael - 2017 IJDMB.pdf">[PDF]</A> 
        <A HREF="paper/Thomas, Sael - 2017 IJDMB.html">[BIBTEX]</A>

<li>Thomas, J., Thomas, S., & Sael, L. (2017). DP-miRNA: an improved prediction of precursor microRNA using deep learning mode. In IEEE International Conference on Big Data and Smart Computing (<a href="http://www.bigcomputing.org/conf2017/">BigComp</a>) <BR>
        <A HREF="paper/thomas, thomas, sael - 2017 - DP-miRNA.pdf">[PDF]</A>  
        <A HREF="paper/thomas2017.html">[BIBTEX]</A>  

<li>Shin, K., Sael, L., & Kang, U. (2017). Fully scalable methods for distributed tensor factorization. IEEE Transactions on Knowledge and Data Engineering (<a href="http://www.computer.org/portal/web/tkde">TKDE</a>), 29(1), 100-113. <BR>
        <A HREF="paper/Shin, Sael, Kang - 2017 - CDTF.pdf">[PDF]</A>  
        <A HREF="paper/shin2017.html">[BIBTEX]</A>  

<li>Jung, J., Park, N., Sael, L., & Kang, U. (2017). BePI: Fast and memory-efficient method for billion-scale random walk with restart. In ACM International Conference on Management of Data (<a href="http://www.sigmod2017.org/">SIGMOD</a>). Raleigh, North Carolina, USA.: ACM Press.  <BR>
        <A HREF="paper/Jung et al. - 2017 - BePI.pdf">[PDF]</A>  
        <A HREF="paper/jung2017.html">[BIBTEX]</A>  
</ul>

<P>
<H4> 2016 </H4> 
<ul>
<li>Thomas, J., & Sael, L. (2016). Maximizing information through multiple kernel-based heterogeneous data integration and applications to ovarian cancer. In 6th International Conference on Emerging Databases (<a href="http://www.emergingdb.org/edb2016">EDB</a>). ACM Press.<BR>
        <A HREF="paper/Thomas, Sael - 2016 - Multiple kernel-based heterogeneous data integration.pdf">[PDF]</A>  
        <A HREF="paper/thomas2016b.html">[BIBTEX]</A> 

<li>Thomas, J., Seo, D., & Sael, L. (2016). Review on graph clustering and subgraph similarity based analysis of neurological disorders. International Journal of Molecular Sciences, 17(6), 862. <BR>
        <A HREF="paper/Thomas, Seo, Sael - 2016 - Review on graph neurological disorders.pdf">[PDF]</A>  
        <A HREF="paper/thomas2016a.html">[BIBTEX]</A> 

<li>Jeon, B., Jeon, I., Sael, L., & Kang, U. (2016). SCouT: Scalable coupled matrix-tensor factorization - algorithm and discoveries. In 32nd IEEE International Conference on Data Engineering (<a href="http://icde2016.fi/">ICDE</a>). Helsinki, Finland.<BR>
        <A HREF="paper/Jeon et al. - 2016 - SCouT.pdf">[PDF]</A>
        <A HREF="paper/jeon2016a.html">[BIBTEX]</A>

<li>Jeon, I., Papalexakis, E. E., Faloutsos, C., Sael, L., & Kang, U. (2016). Mining billion-scale tensors: algorithms and discoveries. The VLDB Journal. <BR>
        <A HREF="paper/Jeon et al. - 2016 - Mining billion-scale tensors algorithms and discoveries.pdf">[PDF]</A>
        <A HREF="paper/jeon2016b.html">[BIBTEX]</A>

<li>Jung, J., Jin, W., Sael, L., & Kang, U. (2016). Personalized ranking in signed networks using signed random walk with restart. In IEEE (Ed.), IEEE International Conference on Data Mining (<a href="http://icdm2016.eurecat.org/">ICDM</a>). Barcelona, Spain. <BR>
        <A HREF="paper/Jung et al. - 2016 - Personalized ranking in signed networks using signed random walk with restart.pdf">[PDF]</A>
        <A HREF="paper/jung2016a.html">[BIBTEX]</A>

<li>Jung, J., Shin, K., Sael, L., & Kang, U. (2016). Random walk with restart on large graphs using block elimination. ACM Transactions on Database Systems (TODS), 41(2), 1-43. <BR>
        <A HREF="paper/Jung et al. - 2016 - Random walk with restart on large graphs using block elimination.pdf">[PDF]</A>
        <A HREF="paper/jung2016b.html">[BIBTEX]</A>
</UL>


<P>
<H4> 2015 </H4> 
<UL>
<LI> Kim, S., Sael, L.*, & Yu, H.* (2015). A mutation profile for top- k patient search exploiting Gene-Ontology and orthogonal non-negative matrix factorization. <B>Bioinformatics</B>, 31(22): 3653-3659. <br> 
        <A HREF="https://academic.oup.com/bioinformatics/article-pdf/31/22/3653/17122597/btv409.pdf">[PDF]</A> 
        <A HREF="paper/kim2015a.html">[BIBTEX]</A>  
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/26209432">[PUBMED]</A>

 <LI> Shin, K., Jung, J., Sael, L., & Kang, U. (2015). BEAR: Block elimination approach for random walk with restart on large graphs. In Proceedings of the 2015 ACM SIGMOD International Conference on Management of Data (<a href="http://www.sigmod2015.org">SIGMOD</a>). International Conference on Management of Data (pp. 1571-1585).  <BR>
        <A HREF="paper/Shin et al. - 2015 - BEAR.pdf">[PDF]</A> 
        <A HREF="paper/shin2015.html">[BIBTEX]</A> 
    
<LI> Sael, L., Jeon, I., & Kang, U. (2015). Scalable tensor mining. Big Data Research, 2(2), 82-86.  <br>
      <A HREF="paper/sael, jeon, kang - 2015 - scalable tensor mining.pdf">[PDF]</A> 
        <A HREF="paper/Sael2015.html">[BIBTEX]</A>
      
<LI> Thomas, J., & Sael, L. (2015). Overview of integrative analysis methods for heterogeneous data. In The 2015 International Conference on Big Data and Smart Computing (<a href="http://www.bigcomputing.org/conf2015">BigComp</a>) (pp. 266-270). <br>
        <A HREF="paper/thomas, sael - 2015 - overview of integrative analysis methods for heterogeneous data.pdf">[PDF]</A> 
        <A HREF="paper/thomas2015.html">[BIBTEX]</A>  
        <!--A HREF="">[PUBMED]</A-->

<LI> Kim, S., Sael, L., & Yu, H. (2015). LMDS-based approach for efficient top-k local ligand-binding site search. International Journal of Data Mining and Bioinformatics, 12(4), 417-433. <br>
        <A HREF="paper/kim, sael, yu - 2015 - lmds-based approach for efficient top-k local ligand-binding site search.pdf">[PDF]</A> 
        <A HREF="paper/kim2015.html">[BIBTEX]</A> 
</UL>


<H4>2014</H4>
<UL>
<LI> Kang, D., Lim, W., Shin, K., Sael, L., & Kang, U. (2014). Data/Feature Distributed Stochastic Coordinate Descent for Logistic Regression. In Proceedings of the 23rd International Conference on Information and Knowledge Management (<a href="http://cikm2014.fudan.edu.cn/">CIKM</a>). Shanghai, China: ACM Press. <BR>
        <A HREF="paper/kang et al. - 2014 - data feature distributed stochastic coordinate descent for logistic regression.pdf">[PDF]</A> 
        <A HREF="paper/dfdscd_supp_theory.pdf">[Supplementary Document]</A> 
        <A HREF="paper/kang2014.html">[BIBTEX]</A>
 
<LI>Xiong, Y., Esquivel-Rodriguez, J., Sael, L., & Kihara, D. (2014). 3D-SURFER 2.0: Web platform for real-time search and characterization of protein surfaces. Methods in Molecular Biology, 1137, 105-117.<BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/xiong2014.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/24573477">[PUBMED]</A>
 
<LI> Kim, S., Sael, L., & Yu, H. (2014). Identifying cancer subtypes based on somatic mutation profile. In Proceedings of the ACM 8th International Workshop on Data and Text Mining in Bioinformatics - <a href="http://www.dtmbio.net/dtmbio2014/">DTMBIO</a> (pp. 19-22). New York, NY, USA: ACM.  <BR>
        <A HREF="paper/Kim, Sael, Yu - 2014 - Identifying cancer subtypes based on somatic mutation profile.pdf">[PDF]</A> 
        <A HREF="paper/Kim2014.html">[BIBTEX]</A> 
</UL> 


<H4>2013</H4>
<UL>
<LI> Pi, J., & Sael, L. (2013). Mass spectrometry coupled experiments and protein structure modeling methods. International Journal of Molecular Sciences, 14(10), 20635-20657. <BR>
        <A HREF="http://www.mdpi.com/1422-0067/14/10/20635/pdf">[PDF]</A>
        <A HREF="paper/Pi2013.html">[BIBTEX]</A>         <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/24132151">[PUBMED]</A>
        
<LI> Kim, S., Sael, L., & Yu, H. (2013). Efficient protein structure search using indexing methods. BMC Medical Informatics and Decision Making, 13(Suppl 1), S8. <BR>
        <A HREF="http://www.biomedcentral.com/content/pdf/1472-6947-13-S1-S8.pdf">[PDF]</A>
        <A HREF="paper/Kim2013d.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/23691543">[PUBMED]</A>
         
<LI>Tang, M., Tan, K. M., Tan, X. L., Sael, L., Chitale, M., Esquivel-rodriguez, J., & Kihara, D. (2014). Graphical models for protein function and structure prediction. In M. Elloumi & A. Y. Zomaya (Eds.), Biological Knowledge Discovery Handbook: Preprocessing, Mining, and Postprocessing of Biological Data (pp. 191-222). Hoboken, New Jersey: John Wiley & Sons, Inc. <BR>
        <A HREF="http://onlinelibrary.wiley.com/doi/10.1002/9781118617151.ch09/pdf">[PDF]</A> 
        <A HREF="paper/Tang2013.html">[BIBTEX]</A>
 
 <LI> Coelho, D., & Sael, L. (2013). Breast and prostate cancer expression similarity analysis by iterative SVM based ensemble gene selection. In Proceedings of the 7th International Workshop on Data and Text Mining in Biomedical Informatics - <a href="http://www.dtmbio.net/dtmbio2013/">DTMBIO</a> (pp. 23-26). San Francisco, USA: ACM Press. 
        <A HREF="paper/Coelho, Sael - 2013 - Breast and prostate cancer expression similarity analysis by iterative SVM based ensembl.pdf">[PDF]</A>
        <A HREF="paper/Coelho2013.html">[BIBTEX]</A>     

<LI>Kim, S., Sael, L., & Yu, H. (2013). Efficient local ligand-binding site search using landmark MDS. In Proceedings of the 7th International Workshop on Data and Text Mining in Biomedical Informatics - <a href="http://www.dtmbio.net/dtmbio2013/">DTMBIO</a> (pp. 13-14). San Francisco, USA: ACM Press.<BR>
        <A HREF="paper/Kim, Sael, Yu - 2013 - Efficient local ligand-binding site search using landmark mds.pdf">[PDF]</A>
         <A HREF="paper/Kim2013b.html">[BIBTEX]</A>

<LI> Kim, S., Sael, L., & Yu, H. (2013). Fast protein 3D surface search. In Proceedings of the 7th International Conference on Ubiquitous Information Management and Communication - <a href="http://www.icuimc.org/2013/index.html">ICUIMC </a>(pp. 1-8). Kota Kinabalu, Malaysia: ACM Press. <BR>
         <A HREF="paper/Kim, Sael, Yu - 2013 - Fast protein 3D surface search.pdf">[PDF]</A> 
        <A HREF="paper/Kim2013a.html">[BIBTEX]</A>         
   
</UL>

<H4>2012</H4>
<UL>
<LI> Mullins, E. a, Starks, C. M., Francois, J. a, Sael, L., Kihara, D., & Kappock, T. J. (2012). Formyl-coenzyme A (CoA):oxalate CoA-transferase from the acidophile Acetobacter aceti has a distinctive electrostatic surface and inherent acid stability. Protein Science, 21(5), 686-696. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Mullins2012.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22374910">[PUBMED]</A>

<LI>Sael, L., Chitale, M., & Kihara, D. (2012). Structure- and sequence-based function prediction for non-homologous proteins. Journal of Structural and Functional Genomics, 13(2), 111-23. <BR>
 <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012b.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22270458">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2012). Detecting local ligand-binding site similarity in nonhomologous proteins by surface patch comparison. <B>Proteins</B>: Structure, Function, and Bioinformatics, 80(4), 1177-1195. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22275074">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2012). Constructing patch-based ligand-binding pocket database for predicting function of proteins. <B>BMC Bioinformatics</B>, 13(Suppl 2), S7. 
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Sael2012a.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/22536870">[PUBMED]</A>
     
<LI>Kim, S., Lee, S., & Yu, H. (2012). Indexing methods for efficient protein 3D surface search. In Proceedings of the ACM sixth international workshop on Data and text mining in biomedical informatics - <A href="http://www.dtmbio.net/dtmbio2012/> DTMBIO</A> (pp. 41-48). <BR>
         <A HREF="paper/kim, sael, yu - 2012 - indexing methods for efficient protein 3d surface search.pdf">[PDF]</A> 
         <A HREF="paper/Kim2012.html">[BIBTEX]</A>
</UL>   

<H4>2011</H4>
<UL>
    <LI> Kihara, D., Sael, L., Chikhi, R., & Esquivel-Rodriguez, J. (2011). Molecular surface representation using 3D Zernike descriptors for protein shape comparison and docking. Current Protein & Peptide Science, 12(6), 520-530. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Kihara2011.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21787306">[PUBMED]</A>
    
<LI> Sael, L. Structural bioinformatics and application to future drug design evolution. Communications of KIISE. 2011; 29:4, 26-32. <BR>
        <!-- <A HREF="">[PDF]</A> --> 

<LI> Chikhi, R., Sael, L., & Kihara, D. Protein binding ligand prediction using moment-based methods  In D. Kihara (Ed.), Protein Function Prediction for Omics Era (pp. 145=163). Dordrecht: Springer Netherlands. 2011. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Chikhi2011.html">[BIBTEX]</A> 
        <A href="http://www.springer.com/biomed/book/978-94-007-0880-8">[ABSTRACT]</A>    

</UL>

<H4>2010</H4>
<UL>
 <LI> Chikhi, R., Sael, L., & Kihara, D. (2010). Real-time ligand binding pocket database search using local surface descriptors. Proteins: Structure, Function, and Bioinformatics, 78(9), 2007-28. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Chikhi2010.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20455259">[PUBMED]</A>
    
<LI> Sael, L., & Kihara, D. (2010). Binding ligand prediction for proteins using partial matching of local surface patches. International Journal of Molecular Sciences, 11, 5009-5026. <BR>
        <A HREF="http://www.mdpi.com/1422-0067/11/12/5009/pdf">[PDF]</A> 
        <A HREF="paper/Sael2010a.html">[BIBTEX]</A>         <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21614188">[PUBMED]</A>
    
<LI> Sael, L., & Kihara, D. (2010). Improved protein surface comparison and application to low-resolution protein structure data. <B>BMC Bioinformatics</B>, 11(Suppl 1), S2. (GIW2010 - BEST PAPER AWARD)  <BR>
        <A HREF="http://www.biomedcentral.com/content/pdf/1471-2105-11-S11-S2.pdf">[PDF]</A>
        <A HREF="paper/Sael2010.html">[BIBTEX]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/21172052">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2010). Characterization and classification of local protein surfaces using self-organizing map. International Journal of Knowledge Discovery in Bioinformatics, 1(1), 32-47. <BR>
        <A HREF="paper/SaelKihara_IJKDB.pdf">[PDF]</A>
        <A HREF="paper/Sael2010b.html">[BIBTEX]</A>
        <A target="blank" href="http://www.igi-global.com/Bookstore/Article.aspx?TitleId=40970">[ABSTRACT]</A>
</UL>

<H4>2009</H4>
<UL>
<LI> La, D., Esquivel-rodriguez, J., Venkatraman, V., Li, B., Sael, L., Ueng, S., Steven, A., Kihara, D. (2009). 3D-SURFER: Software for high-throughput protein surface comparison and analysis. Bioinformatics, 25(21), 2843-4. <BR>
        <A HREF="http://bioinformatics.oxfordjournals.org/content/25/21/2843.full.pdf+html">[PDF]</A> 
        <A HREF="paper/La2009.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/19759195">[PUBMED]</A>

<LI> Venkatraman, V., Yang, Y. D., Sael, L., & Kihara, D. (2009). Protein-protein docking using region-based 3D Zernike descriptors. BMC Bioinformatics, 10, 407. <BR>
        <A href="http://www.biomedcentral.com/content/pdf/1471-2105-10-407.pdf">[PDF]</A>
        <A HREF="paper/Venkatraman2009.html">[BIBTEX]</A>
        <A href="paper/lzerd_Suppl.pdf">[Suppl. Data]</A>
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/20003235">[PUBMED]</A>
    
<LI>Venkatraman, V., Sael, L., & Kihara, D. (2009). Potential for protein surface shape analysis using spherical harmonics and 3D Zernike descriptors. Cell Biochemistry and Biophysics, 54(1-3), 23-32. <BR>
        <!-- <A HREF="paper/">[PDF]</A> --> 
        <A HREF="paper/Venkatraman2009a.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/19521674">[PUBMED]</A>

<LI> Sael, L., & Kihara, D. (2009). Protein surface representation and comparison: New approaches in structural proteomics. In J. Y. Chen & S. Lonardi (Eds.), Biological Data Mining (pp. 89-109). USA: Chapman & Hall/CRC Press. <BR>
        <A HREF="paper/Sael, Kihara - Biological Data Mining 2009.pdf">[PDF]</A> 
       <A HREF="paper/Sael2009.html">[BIBTEX]</A> 
        <A href="http://www.amazon.com/Biological-Mining-Chapman-Knowledge-Discovery/dp/1420086847/">[ABSTRACT]</A>    

</UL>
<H4>2008</H4>
<UL>
<LI> Sael, L., La, D., Li, B., Rustamov, R., & Kihara, D. (2008). Rapid comparison of properties on protein surface. <B>Proteins</B>: Structure, Function, and Bioinformatics, 73(1), 1-10.  <BR>
        <!-- <A HREF="">[PDF]</A> --> 
        <A HREF="paper/Sael2008.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/18618695">[PUBMED]</A>
    
<LI>Sael, L., Li, B., La, D., Fang, Y., Ramani, K., Rustamov, R., & Kihara, D. (2008). Fast protein tertiary structure retrieval based on global surface shape similarity. <B>Proteins</B>: Structure, Function, and Bioinformatics, 72(4), 1259-1273. <BR>
        <!-- <A HREF="">[PDF]</A> --> 
        <A HREF="paper/Sael2008a.html">[BIBTEX]</A> 
        <A HREF="http://www.ncbi.nlm.nih.gov/pubmed/18361455">[PUBMED]</A>
        <BR>
        <BR>
</UL>
</section> <!-- publications -->
<P>


<!-- END CONTENT-->

</div> <!-- /#content -->
</div> <!-- /.container -->
</div> <!-- /#main -->

<!--FOOTER INCLUDE-->
<object name="header" width="100%" height="30pt" type="text/html" data="footer.html"></object>


</BODY></HTML>










